xbmc/settings/AdvancedSettings.cpp

```diff
@@ -280,6 +280,7 @@ void CAdvancedSettings::Initialize()
   m_strMusicLibraryAlbumFormat = "";
   m_prioritiseAPEv2tags = false;
   m_musicItemSeparator = " / ";
+  m_musicArtistSeparators = { ";", ":", "|", "#", " feat. ", " ft. " };
```

Comment:
Could be made case sensitive to avoid that maybe? Ft. = foot and ft. = featuring? who knows!